src/client/entity-editor/common/entity.js

```diff
@@ -19,49 +19,52 @@
 
 // @flow
 
-import Icon from 'react-fontawesome';
+// import Icon from 'react-fontawesome';
 import React from 'react';
 import {genEntityIconHTMLElement} from '../../helpers/entity';
 
 
 type EntityProps = {
 	disambiguation?: ?string,
-	link?: string | false,
 	text: string,
 	type: string,
 	unnamedText?: string
 };
 
 function Entity(
-	{disambiguation, link, text, type, unnamedText}: EntityProps
+	{disambiguation, id, text, type, unnamedText}: EntityProps
 ) {
-	const nameComponent = text || <i>{unnamedText}</i>;
+	const url = type === 'Area' ?
+		`//musicbrainz.org/area/${id}` :
+		`/${type.toLowerCase()}/${id}`;
+	// const nameComponent = text || <i>{unnamedText}</i>;
 	const contents = (
 		<span>
 			{
 				type && genEntityIconHTMLElement(type)
 			}
 			{' '}
-			{nameComponent}
-			{' '}
+			<a href={url} rel="noopener noreferrer" target="_blank">
```

Comment:
For me, clicking on one of these links only selects the item and the dropdown closes, instead of opening the link.
I think you'll need to bind an onclick handler to your <a> tag and use `event.preventDefault()` inside the handler for it to work.

I should note that one the dropdown is closed, I can click the link successfully.

Reply:
one the dropdown is closed, I can click the link successfully.... yes that is what I have tried to do. And okay I will try to bind the onclick handler.